src/DotNetLightning.Core/Transactions/Transactions.fs

```diff
@@ -297,6 +298,13 @@ module Transactions =
         [<Literal>]
         let OFFERED_HTLC_SCRIPT_WEIGHT = 133uy
 
+    let CreateTransactionBuilder (network: Network) =
```

Comment:
`let` function must be camelCase. https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#use-camelcase-for-module-bound-public-functions

Or in this case, you can define as an extension method to the `Network` , in that case you should define in `Utis/NBitcoinExtensions.fs`

Reply:
why do you want to differ from the Microsoft's .NET API design guidelines? In them, methods are always PascalCase, not camelCase.

Reply:
> I can or I should? Cause I'm not a fan of extension methods, so if the former, I'll leave it as is.

I meant "can", No problem.

Reply:
>  (so I guess adding private or internal is better BTW)

Sure sorry I make it private.

> If we want to expose function to C#, then we can use

Why go into such annoying standard and not just adopt the same standards as C#? I only use camelCase myself for functions when they are local functions (functions inside of functions) or when the function is passed as a parameter. Personal preference though...

Reply:
> Why go into such annoying standard and not just adopt the same standards as C#? I only use camelCase myself for functions when they are local functions (functions inside of functions) or when the function is passed as a parameter. Personal preference though...

I think it is kind of bikeshedding to chose which style, but consistency matters. So please use camelCase in this code base.